src/extensions/default/BrambleUrlCodeHints/style.less

```diff
@@ -6,6 +6,11 @@
     width: 320px;
     height: auto;
     background-color: black !important;
+    transform: rotateY(180deg);
```

Comment:
Move this unprefixed version to the end of the list so that it isn't overridden by a prefixed version.